### PR TITLE
Cleanup lines data

### PR DESCRIPTION
**Pull request by kmj94**
Big picture, all the daily files were deleted, and instead of nba_lines, nba_lines_22_23, nba_lines_current, and nba_lines_new folders, we now just have nba_lines_historical and nba_lines_current. For the new folders, they each have 3 files, one for each bet type (spread, total, moneyline). At the first run for each day, we now take the data from the current folder and append it to the data in the historical folder, then overwrite current with the new day's lines.

The general structure will be
- During the first run of the day, we use 00_store_historical_nba_lines.R to take the final lines from the prior day's games and append it to our base historical files, and overwrite the historical files each day. The change in this script is that we now append to the historical file as opposed to writing out new files daily, you can view the changes here https://github.com/jimtheflash/gambling_stuff/commit/bb3e266a937de9757a01c22d1fda1ff08a1d9036

- During every intra day run, we pull in the latest lines from 00_get_current_nba_lines.R , and write out to the nba_current_..csv files. Nothing changed here, although once we get closer to the season there will be a couple of changes to make regarding table schema structure, I just want to be able to read in actual lines to make sure the code will run as expected, so holding off for now.

- During every intra day run, we then use the 01_tidyup_nba_lines.R script. The new version is here https://github.com/jimtheflash/gambling_stuff/blob/cleanup_lines_data/first_basket/intra_day_runs/01_tidyup_nba_lines.R, while the old version is here https://github.com/jimtheflash/gambling_stuff/blob/main/first_basket/intra_day_runs/01_tidyup_nba_lines.R. This had the biggest changes, the old version took in the daily files from all the different folders and did the tidyup work every run. As a part of this PR I basically saved off the work from the old script into the new historical files and now the only thing that needs to be done in this script is pivoting the lines from wide to long and then joining them together to get a single row for every game/team. This results in almost the same output that feeds in to the model as before, except now it is a little more cleaned up and we don't have extra columns for the quarterly box score totals, etc.

So to summarize, the lines data will now live in two places
- "Raw" data that is stored in ./data/nba_lines_historical/nba_historical_(bettype).csv' and ./data/nba_lines_current/nba_current_(bettype).csv', with one row for each game
- "Feature" data that is stored in './data/02_curated/nba_first_to_score/historical_lines.csv' and './data/02_curated/nba_first_to_score/current_lines.csv' that is used as model inputs

### CODE REVIEW

**Comment by kmj94** [comment]
@jimtheflash yeah not too worried about the code itself, more just wanted to explain my process and make sure that we're all good with the way it would be structured going forward. I think the links in the PR description should get you to the script changes
